composer.json

```diff
@@ -35,9 +35,9 @@
         "laravel/framework": "^8.83.14",
         "livewire/livewire": "^2.7",
         "lychee-org/nestedset": "^6",
-        "lychee-org/php-exif": "^0.7.5",
+        "lychee-org/php-exif": "^v0.7.6",
```

Comment:
Why do we suddenly need the `v` in front of the version?

Reply:
That's interesting, because on my setup PhpStorm doesn't mind. Yes, I would remove it, because we don't have it anywhere else. I am a huge fan of keeping and making things consistent.